woocommerce/woocommerce
# Playwright 1.32.3 update

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Updates Playwright to 1.32.3 and adds `pnpm test:e2e-pw-ui` for running Playwright in UI mode.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure e2e tests pass on CI
2. Pull branch
3. `pnpm env:start`
4. `pnpm test:e2e-pw-ui` -- make sure tests start and Playwright UI window appears
5. `pnpm test:e2e-pw` -- ensure tests pass
